src/components/todoItem.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './todoItem.css';
+
+export default class TodoItem extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  removeTodo(id) {
+    this.props.removeTodo(id);
+  }
+
+  render() {
+    return (
+      <div className="todoWrapper">
+        <button className="removeTodo" onClick={(e)=> this.removeTodo(this.props.id) }>remove</button>{this.props.todo.text}
```

Comment:
using a lambda is a good way to only bind the function you're calling to the element in the render method that you're using to call it with. However, a pattern that I like, (since you have a constructor) is to bind the handlers in the constructor . You also are passing in "e" and not using it. I'd advice against this in the future.